flutter_local_notifications/android/src/main/java/com/dexterous/flutterlocalnotifications/FlutterLocalNotificationsPlugin.java

```diff
@@ -179,6 +183,19 @@ private static Notification createNotification(Context context, NotificationDeta
             builder.setWhen(notificationDetails.when);
         }
 
+        if (notificationDetails.isFullScreen){
+            builder.setFullScreenIntent(pendingIntent, true);
+
+            // wake device
+            PowerManager powerManager = (PowerManager) context.getSystemService(Context.POWER_SERVICE);
```

Comment:
Is the rest of the code here actually needed? From what I'm reading in the [docs](https://developer.android.com/training/notify-user/time-sensitive), there's no mention of it being needed

Reply:
> Is the rest of the code here actually needed? From what I'm reading in the [docs](https://developer.android.com/training/notify-user/time-sensitive), there's no mention of it being needed

I'll verify, but I think it is required for the screen to be able to turn-on and stay on.

Reply:
I think you're right.